lib/model/folder_sendrecv.go

```diff
@@ -611,7 +610,7 @@ func (f *sendReceiveFolder) handleDir(file protocol.FileInfo, snap *db.Snapshot,
 			err = f.deleteItemOnDisk(curFile, snap, scanChan)
 		}
 		if err != nil {
-			f.newPullError(file.Name, err)
+			f.newPullError(file.Name, fmt.Errorf("handling dir (conflict move): %w", err))
```

Comment:
Actually the conflict move context only applies to the error on L606, L610 just deletes stuff. Maybe just drop that context in this PR.